src/evaluate/module.py

```diff
@@ -441,7 +459,12 @@ def compute(self, *, predictions=None, references=None, **kwargs) -> Optional[di
 
             inputs = {input_name: self.data[input_name] for input_name in self._feature_names()}
             with temp_seed(self.seed):
-                output = self._compute(**inputs, **compute_kwargs)
+                config_state = deepcopy(self.config)
+                self.config.update(compute_kwargs)
+
+                output = self._compute(**inputs)
+
+                self._module_info.config = config_state
```

Comment:
(nit)
```suggestion
                self.config.update(compute_kwargs)
                try:
                    output = self._compute(**inputs)
                finally:
                    self._module_info.config = config_state
```

Reply:
That change got lost in a stash! Thank you!